types/Settings.d.ts

```diff
@@ -170,6 +170,9 @@ export class Settings {
   get language(): string;
   set language(value: string);
 
+  /** Whether mapUrls (the wfmap.gz files from the CDN) are loaded  */
+  get loadMapUrls(): string;
```

Comment:
```suggestion
  get loadMapUrls(): boolean;
```

Should this be a Boolean too?